www/src/components/feedback-widget/feedback-form.js

```diff
@@ -71,6 +71,8 @@ const textareaStyles = {
   height: `5.5rem`,
   mt: 1,
   mb: 4,
+  lh: `1.25rem`,
```

Comment:
Oh wow I didn't know that `lh` works. Nice!
I think we should be able to use `dense` (matching `1.25rem`) from the theme tokens here though?

```
lh: `dense`
```